client/client.go

```diff
@@ -56,8 +56,8 @@ type Client struct {
 // MakeClient make client
 func MakeClient(apiKey, secretKey, endpoint string) (*Client, error) {
 
-	if apiKey == content.NONE || secretKey == content.NONE {
-		return &Client{}, fmt.Errorf(content.ERROR_MISSING_API_KEY_SECRET_KEY)
+	if apiKey == "None" || secretKey == "None" || apiKey == "" || secretKey == "" {
+		return nil, NewError("Missing API key or/and Secret key. Please run 'coreo configure' to configure them.")
```

Comment:
Shouldnt this be content.ERROR_MISSING_API_KEY_SECRET_KEY ??? or something similar?